adapters/criteo/criteo.go

```diff
@@ -25,9 +26,24 @@ type ExtPrebid struct {
 	NetworkName string              `json:"networkName"`
 }
 
+type CriteoExt struct {
+	Igi []*CriteoExtIgi `json:"igi,omitempty"`
+}
+
+type CriteoExtIgi struct {
+	ImpId string          `json:"impid"`
+	Igs   []*CriteoExtIgs `json:"igs"`
+}
+
+type CriteoExtIgs struct {
+	ImpId  string          `json:"impid"`
```

Comment:
This field is not set in the newly added code. Is it part of the spec or a copy/paste error?

Reply:
It is part of the ortb schema but we do not use it, I removed it.